### ISSUE TITLE
setPedDoingGangDriveby in onClientPlayerWeaponFire crashes game

### ISSUE THREAD

**Issue by Fernando-A-Rocha** [multitheftauto/mtasa-blue#1913]
In a function handled by event **onClientPlayerWeaponFire**
Doing **setPedDoingGangDriveby**(localPlayer, false) will **_crash_** the player's game
Happens all the time

To try this out for yourself:
```lua
local count = 0
function testBug(weapon, ammo, ammoInClip, hitX, hitY, hitZ, hitElement )
	if source == localPlayer then
        
        if count == 5 then
            -- trigger it; Game will crash
            if isPedDoingGangDriveby(localPlayer) then
                setPedDoingGangDriveby(localPlayer, false)
            end
        end
        
        count = count + 1
	end
end
addEventHandler ( "onClientPlayerWeaponFire", root, testBug )
```

**Comment by Fernando-A-Rocha** [comment]
Version = 1.5.8-release-20698.3.005
Time = Wed Dec  9 12:22:24 2020
Module = C:\Program Files (x86)\Rockstar Games\GTA San Andreas\proxy_sa.exe
Code = 0xC0000005
Offset = 0x0022D615

EAX=00000000  EBX=00000003  ECX=00000000  EDX=FFFFFFFF  ESI=23B058A0
EDI=0000004A  EBP=1DC5B190  ESP=0177FCA4  EIP=0062D615  FLG=00210206
CS=0023   DS=002B  SS=002B  ES=002B   FS=0053  GS=002B

**Comment by Dutchman101** [comment]
Incorrect animation association pointer in ECX. ECX value is always 0.

Red dot is where it crashes:
![image](https://user-images.githubusercontent.com/923242/101756836-5cd3f200-3ace-11eb-9f10-c2a28ef8c87a.png)

**Comment by Tru-KaaN** [comment]
worth to inform you that using even a 0ms timer prevents the crash, example;
```lua
addEventHandler("onClientPlayerWeaponFire", localPlayer, function()
    if isPedDoingGangDriveby(localPlayer) then 
        setTimer(setPedDoingGangDriveby, 0, 1, localPlayer, false)
    end
end)
```

**Comment by Pirulax** [comment]
Yes, because even timers with lower interval than the frametime (delta between frames) are executed on the next frame, so basically it seems like the animation isn't loaded at that point, or something, but it is in the next frame.
